src/app/tables/table.component.ts

```diff
@@ -13,6 +13,9 @@ export class EstuaryTableComponent implements OnChanges {
   @Input() title: string;
   // An optional title link
   @Input() titleLink: string;
+  // Only necessary if there is no title ; the name of the CSV file defaults to the
+  // title name automatically
+  @Input() csvName: string;
```

Comment:
Optional: I think the name `csvName` is fine, but I wonder if `csvFileName` is easier to understand. I'll leave it up to you.